WordPress/gutenberg
# [RNmobile] Improve "take video" icon

## Description
This PR improves the "Take a Video" icon we're using on the media selection bottom sheet. So far, we've been using the "camera" icon, the very same as the "Take a Photo" icon, which is apparently less than ideal to be the same.

<img width="381" alt="Screenshot 2020-03-16 at 15 26 23" src="https://user-images.githubusercontent.com/1032913/76765000-6d08ac00-679e-11ea-8843-3c27599d76ac.png">

## How has this been tested?
Use this gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2024

To test:
1. Run the native mobile demo app and add a Media&Text block
2. Tap on the "Add image or video" button in the media area of the block
3. Notice the video-alt2 icon being used for the "Take a video" option

## Types of changes
1. Introducing the "video-alt2" icon, ported from the corresponding Dashicon https://github.com/WordPress/dashicons/blob/master/sources/svg/video-alt2.svg. Adding you @youknowriad as a reviewer on this PR for the web-related changes.
2. I'm not sure if I followed the proper procedure to port the icon. I copy/pasted the SVG path but I also moved the icon a bit to the right of its otherwise horizontal center, as it wasn't visually balanced in the very center. Used Inkscape for that movement. @jasmussen maybe you have some clue here whether this looks OK or not, or perhaps there's a better way of porting the icon from Dashicons.
3. Using the new icon on the native mobile bottom sheet when selecting media

## Checklist:
